content/blog/2020/release-0.2.0.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Release v0.2.0"
+description: "Announcing the v0.2.0 release of BDK"
+author: "Alekos Filini"
+date: "2020-12-21"
+tags: ["rust", "release"]
+hidden: true
+draft: false
+---
+
+After a very long wait, a new release of BDK is finally out, with many exciting new features, bug fixes and overall improvements. This release marks the beginning of our new regular [release schedule][release_schedule], which will see us pushing
+out a new release every four weeks. We think this is a good compromise to ensure that developers using BDK have access to all the new features and fixes as soon as possible, at least while the library is still evoling very fast as it is
+right now. After `v1.0.0` we will increase this time to a more relaxed 6 weeks.
+
+You can also find the [full changelog][changelog] on GitHub for this release.
+
+## What's new in v0.2.0
+
+While this release is technically the second ever release of BDK, it almost feel like it is the first considering the sheer amount of new things being added: we can't really explain every single one of them here in detail, but we'll try our best
+to give a quick overview of some of the new features you could find useful in your projects.
```

Comment:
this could be simplified a bit like this: "considering the sheer amount of new things being added.  We don't have room here to explain every new feature in detail, but below is a quick overview of some you could find useful in your projects."